Modules/Microsoft365DSC/DSCResources/MSFT_SPOAccessControlSettings/MSFT_SPOAccessControlSettings.psm1

```diff
@@ -413,7 +429,8 @@ function Test-TargetResource
             'DisallowInfectedFileDownload', `
             'ExternalServicesEnabled', `
             'EmailAttestationRequired', `
-            'EmailAttestationReAuthDays')
+            'EmailAttestationReAuthDays',
+        'ConditionalAccessPolicy')
```

Comment:
Just one minor thing: Could you please make sure the indentation is the same as the ones above?

Reply:
I'm so sorry that my changes had so many issues. I have corrected the mentioned indentation.